edge-modules/azure-monitor/src/Program.cs

```diff
@@ -85,24 +84,6 @@ static async Task<int> MainAsync()
             return 0;
         }
 
-        //TODO: we don't use the module twin for anything, don't bother getting tags?
-        static async Task<Option<SortedDictionary<string, string>>> GetAdditionalTagsFromTwin(ModuleClient moduleClient)
-        {
-            Twin twin = await moduleClient.GetTwinAsync();
-            TwinCollection desiredProperties = twin.Properties.Desired;
-            LoggerUtil.Writer.LogInformation($"Received {desiredProperties.Count} tags from module twin's desired properties that will be added to scraped metrics");
-
-            const string additionalTagsPlaceholder = "additionalTags";
-            Dictionary<string, string> deserializedTwin = JsonConvert.DeserializeObject<Dictionary<string, string>>(twin.Properties.Desired.ToJson());
-            if (deserializedTwin.ContainsKey(additionalTagsPlaceholder))
-            {
-                return Option.Some<SortedDictionary<string, string>>(ParseKeyValuePairs(deserializedTwin[additionalTagsPlaceholder], LoggerUtil.Writer, true));
```

Comment:
Unfortunately the logic for ```ParseKeyValuePairs``` is still in the module. I think we should revert the change that remove ```additionalTags``` and leave it in for now.

After the module is checked in initially we can remove this functionality in a separate PR.